DOCS-9779 added links to runtime manager alerts page (#330)
mulesoft/docs-api-manager: modules/ROOT/pages/using-api-alerts.adoc

* The API returns a specified HTTP error code.
* The API response time exceeds a certain amount.

> xref:runtime-manager::alerts-on-runtime-manager.adoc[Runtime Manager alerts] differ in nature from API alerts. Runtime Manager flags CPU usage, secure data gateway up/down, and deployment success/failure, to name a few. API alerts, on the other hand, relate to an API request or response.

Reviewer: The topic you're linking to is called "Alerts", so maybe move Runtime Manager out of the link?

Author: Done! Thanks!

> xref:runtime-manager::alerts-on-runtime-manager.adoc[Runtime Manager alerts] differ in nature from API alerts. Runtime Manager flags CPU usage, secure data gateway up/down, and deployment success/failure, to name a few. API alerts, on the other hand, relate to an API request or response.

Reviewer: ```suggestion
xref:runtime-manager::alerts-on-runtime-manager.adoc[Alerts] in Runtime Manager differ from API alerts. For example, Runtime Manager flags CPU usage, secure data gateway up and down, and deployment success and failure. API alerts relate to an API request or response.
```

Author: Done! Thanks!
